templates/includes/sidebar.html

```diff
@@ -1,4 +1,11 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li><a href="{% url "taxi:driver-detail" pk=user.id %}">User: {{ user.username }}</a></li>
+    <li><a href="{% url "logout" %}">Logout</a></li>
+  {% else %}
+    <li><a href="{% url "login" %}">Login</a></li>
+  {% endif %}
+  <br>
```

Comment:
Consider using CSS for spacing instead of `<br>` tags for better control over the layout and to maintain a clean HTML structure.